src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -299,6 +304,10 @@ def _gen_paths(cls, all_endpoints):
                 if summary:
                     methods[method.lower()]['summary'] = summary
 
+                # if method.lower() in ['get', 'delete']:
```

Comment:
What is the purpose of this comment?
Can it be erased?

Reply:
Yes, no need of it. Will erase it